src/backend/marsha/core/tests/test_api_video.py

```diff
@@ -154,6 +156,57 @@ def test_api_video_read_detail_student_other_video(self):
         )
         self.assertEqual(response.status_code, 403)
 
+    def test_api_video_read_detail_student_join_forced(self):
+        """Student users should be allowed to read jitsi info
```

Comment:
I think he will also have medialive info and we don't want it. Can you add medialive in the `live_info` when you create your video to test it ?